Android.bp

```diff
@@ -44,7 +44,8 @@ android_app {
     required: [
         "LocalContactsBackup",
         "privapp_whitelist_com.stevesoltys.backup",
-        "com.stevesoltys.backup_whitelist"
+        "com.stevesoltys.backup_whitelist",
+        "SeedvaultOverlay",
```

Comment:
This should not be added here. Not everybody that ships Seedvault would want to set it as the default, for example LineageOS still defaults to the `gms` provider and instead uses a patch to add a selector to Settings.

* overlay: https://review.lineageos.org/c/LineageOS/android_vendor_lineage/+/288544
* Settings patch: https://review.lineageos.org/c/LineageOS/android_packages_apps_Settings/+/288543